client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -59,7 +60,7 @@ export const generatePath: GeneratePathFunction = ( props, additionalArgs = {} )
 		plan: props.selectedPlan,
 	};
 
-	if ( props.isInSignup || 'customerType' in additionalArgs ) {
+	if ( props.isInSignup || 'customerType' in additionalArgs || props.isStepperUpgradeFlow ) {
```

Comment:
Allows us to toggle "Pay monthly/Pay annually" options without breaking and redirecting away from the plans page.